libbpf-cargo/src/btf/btf.rs

```diff
@@ -227,6 +227,43 @@ impl<'a> Btf<'a> {
         })
     }
 
+    /// Returns an expression that evaluates to the Default value
+    /// of a type(typeid) in string form.
+    ///
+    /// To be used when creating a impl Default for a structure
+    ///
+    /// Rule of thumb is `ty` must be a type a variable can have.
+    ///
+    /// Type qualifiers are discarded (eg `const`, `volatile`, etc).
+    pub fn type_default(&self, type_id: u32) -> Result<String> {
+        let stripped_type_id = self.skip_mods_and_typedefs(type_id)?;
+        let ty = self.type_by_id(stripped_type_id)?;
+
+        Ok(match ty {
+            BtfType::Void => "std::ffi::c_void::default()".to_string(),
+            BtfType::Int(_) => format!("{}::default()", self.type_declaration(type_id)?),
```

Comment:
Nit: can save a few cycles 
```suggestion
            BtfType::Int(_) => format!("{}::default()", self.type_declaration(stripped_type_id)?),
```